content/SDKs/javascript/api-reference.mdx

```diff
@@ -260,6 +263,150 @@ const deletedCount = await context.vector.delete('product-descriptions', 'id1');
 console.log(`Deleted ${deletedCount} vector(s)`);
 ```
 
+### Object Storage
+
+The Object Storage API provides a way to store and retrieve objects (files, documents, media) with support for public URL generation. It is accessed through the `context.objectstore` object.
+
+#### `get(bucket: string, key: string): Promise<DataResult>`
+
+Retrieves an object from the object storage.
+
+##### Parameters
+
+- `bucket`: The bucket to get the object from
+- `key`: The key of the object to get
+
+##### Return Value
+
+Returns a Promise that resolves to a DataResult object. The DataResult has an `exists` boolean property and a `data` property containing the object data if found.
+
+##### Example
+
+```typescript
+// Retrieve an object from object storage
+const result = await context.objectstore.get('user-uploads', 'profile-123.jpg');
+if (result.exists) {
+  // Access data using the Data interface methods
+  const imageData = await result.data.binary();
+  console.log(`Image size: ${imageData.byteLength} bytes`);
+} else {
+  console.log('Image not found');
+}
+```
+
+#### `put(bucket: string, key: string, data: DataType, params?: ObjectStorePutParams): Promise<void>`
+
+Stores an object in the object storage.
+
+##### Parameters
+
+- `bucket`: The bucket to put the object into
+- `key`: The key of the object to put
+- `data`: The data to store (can be ArrayBuffer, string, or other DataType)
+- `params` (optional): Additional parameters for the object. The following properties map to canonical HTTP headers: `contentType` (Content-Type), `contentEncoding` (Content-Encoding), `cacheControl` (Cache-Control), `contentDisposition` (Content-Disposition), `contentLanguage` (Content-Language), and `etag` (ETag). All other properties are added as generic metadata to the underlying storage provider (if supported). The `contentType` parameter is optional - if not provided, the content type will be automatically detected based on the data type (e.g., objects will be set to `application/json`). If the content type cannot be determined, it defaults to `application/octet-stream`.
```

Comment:
the values for the headers should be in the correct format such as "Content-Type", "Content-Length" instead of the proper case